client/src/app/auth/auth.service.ts

```diff
@@ -40,7 +40,7 @@ export class AuthService {
     });
   }
 
-  private setUser(userData: UserModel): void {
+  public setUser(userData: UserModel): void {
```

Comment:
I dont like that your code depends on tests. Maybe it's not so critical in this case but it's better to avoid this

Reply:
fixed that in new commit